### PR TITLE
Adding comments to forge tests

### PR DESCRIPTION
**Pull request by ith-harvey**
Added comments to all forge tests detailing their purpose and any reverts if thrown.

### CODE REVIEW

File: src/_test/ERC20PoolBorrow.t.sol [hunk 1]
```diff
@@ -241,6 +246,9 @@ contract ERC20PoolBorrowTest is DSTestPlus {
         assertEq(pool.totalDebt(), 30000.273023081959005000 * 1e45);
     }
 
+    // @notice: lender deposits 200,000 quote accross 3 buckets
+    // @notice: borrower successfully borrows 100,000 quote
+    // @notice: borrower2 unsuccessfully attempts to borrow pool is undercolalteralized
```
Comment (EdNoepel) [inline src/_test/ERC20PoolBorrow.t.sol]: sp: undercollateralized

File: src/_test/ERC20PoolBorrow.t.sol [hunk 2]
```diff
@@ -309,10 +318,14 @@ contract ERC20PoolBorrowTest is DSTestPlus {
...
         borrower.addCollateral(pool, 100 * 1e18);
-        // should not revert when borrower takes a loan on 100_000 DAI
+        // should not revert when borrower takes a loan on 10_000 DAI
         borrower.borrow(pool, 1_000 * 1e18, 13.537 * 1e18);
     }
 
+    // @notice: lender deposits 150,000 quote accross 3 buckets
+    // @notice: borrower borrows 50,000 quote
+    // @notice: borrower2 borrows 50,000 quote
+    // @notice: lender deposits 1,000 quote
```
Comment (EdNoepel) [inline src/_test/ERC20PoolBorrow.t.sol]: Not sure these comments are valuable in such a test.  May be more useful to see something like "lender deposits into multiple buckets, price pointers validated after debt is drawn from multiple borrowers".

**Review by EdNoepel — COMMENTED** [review]
Current changes look good.

**Review by MikeHathaway — APPROVED** [review]
LGTM!

**Review by EdNoepel — APPROVED** [review]
LGTM

**Pull request merged**